docs/source/pillars/information_governance.md

```diff
@@ -2,137 +2,258 @@
   - Mandatory
-* - You should use codified change processes when altering your policies and standard operating procedures.
-  -
-  - Recommended
-* - You could use version control to track changes to their policies and processes.
+* - You must ensure all policies & standard operating procedures relevant to the TRE organisation are version controlled and have codified change processes.
   - Version control includes recording dates of changes, person responsible for carrying out changes, and summary of changes.
-  - Optional
+  - Mandatory
 ```
 
-### Issue management
+**Quality Management Reporting:** The generation and dissemination of reports or dashboards that provide insights and metrics on the performance and effectiveness of quality management processes and activities.
 
-_The ability of a TRE operator to track deviations from stated policies._
+```{list-table}
+:header-rows: 1
+:name: tab-ig-quality-management-reporting
+
+* - Statement
+  - Guidance
+  - Importance
+* - You must measure the performance of information governance within the TRE with regular reporting available to top management.
+  - This may include reports and dashboards showing security incidents, quality management deviations and audit findings.
+  - Mandatory
+```
+
+**Internal Audit:** An independent evaluation process within the TRE organisation that assesses and improves its internal controls, risk management, and governance.
 
 ```{list-table}
 :header-rows: 1
-:name: tab-issue-management
+:name: tab-ig-internal-audit
 
 * - Statement
   - Guidance
   - Importance
-* - You must have a clear process in place for addressing any activity that deviates from your policies and standard operating procedures.
-  - This can include measures like triage analysis and a process for updating policies.
+* - You must audit your TRE organisation against relevant requirements and standards
+  - If you are publicly accredited against a standard, for instance ISO27001, DSPT, CE+ etc., you must have processes in place to ensure you remain compliant.
   - Mandatory
-* - You must have methods in place to record progress in resolving issues with, and deviations against, your policies.
-  -
+* - You must report on and share outcomes of each audit of your TRE organisation with the required bodies.
+  - This may include regulatory bodies or the organisations that manage accreditations you have
+  - Mandatory
+```
+
+**Supplier Management and Monitoring:** A structured approach to managing and monitoring relationships with external suppliers, vendors and contractors, including selection, contract management and compliance oversight.
+
+```{list-table}
+:header-rows: 1
+:name: tab-ig-supplier-management-monitoring
+
+* - Statement
+  - Guidance
+  - Importance
+* - You must ensure that suppliers, contractors and sub-contractors align with the security requirements of you TRE.
```

Comment:
Suppliers of what? Does this apply to eg. cleaners?

Reply:
Maybe IT suppliers? @machintim what do you think?

Reply:
```suggestion
* - You must ensure that suppliers, contractors and sub-contractors with access to your TRE align with your security requirements.
```